### PR TITLE
PEAR-1819: Handle exceptions related to login window

### PR DESCRIPTION
**Pull request by paribartandhakal**
## Description
Adding a try-catch block.

## Checklist
- [ ] Added proper unit tests
- [ ] Left proper TODO messages for any remaining tasks
- [ ] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

**Comment by wteouchicago** [comment]
this worked for me. I didn't see the errors in the console and I was able to log in, download some CA data, and log out without anything unusual happening. 👍

File: packages/portal-proto/src/features/layout/auth/openAuthWindow.ts [hunk 1]
```diff
@@ -13,20 +13,21 @@ const openAuthWindow = (): Promise<unknown> => {
...
+            if (win.document.URL.includes("error=401")) {
+              reject("login_error");
+              return;
+            }
+            resolve("success");
           }
-          // Resolve that we have something good
-          resolve("success");
+        } catch (err) {
+          // just want to catch it and not reject it
```
Comment (amlehman) [inline packages/portal-proto/src/features/layout/auth/openAuthWindow.ts]: Could you add a little more context/reasoning to this comment?

**Review by amlehman — APPROVED** [review]

**Review by craigrbarnes — CHANGES_REQUESTED** [review]
Looks OK, but I agree with Amy's request for more context in the comment.

**Comment by paribartandhakal** [comment]
> Looks OK, but I agree with Amy's request for more context in the comment.

@amlehman @craigrbarnes  I just updated it. Please let me know if it's clear now.

**Review by craigrbarnes — APPROVED** [review]

**Pull request merged**